astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -32,3 +34,20 @@
         wildcard_match=True,
         bucket_name="sample-bucket",
     )
+
+    check_if_key_with_size_without_wildcard = S3KeySizeSensorAsync(
+        task_id="check_if_key_with_size_without_wildcard",
+        bucket_key="sample_key.txt",
+        wildcard_match=False,
+        bucket_name="test-team-providers-ankit",
+    )
+
+    check_if_key_with_size_with_wildcard = S3KeySizeSensorAsync(
+        task_id="check_if_key_with_size_with_wildcard",
+        bucket_key="sample*",
+        wildcard_match=True,
+        bucket_name="test-team-providers-ankit",
```

Comment:
Use environment variable here instead of hardcoding the bucket_name.Refer to big query example DAG for reference.

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/86/commits/3d9af809ca7095a0ddcded6f686ca64988380ace)